content/en/docs/tasks/traffic-management/traffic-shifting/test.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+
+# @setup profile=default
+
+kubectl label namespace default istio-injection=enabled --overwrite
+startup_sleep_sample # needed for sending test requests with curl
+
+# launch the bookinfo app
+startup_bookinfo_sample
+
+# Verification util
+
+SLEEP_POD=$(kubectl get pod -l app=sleep -n default -o 'jsonpath={.items..metadata.name}')
+INGRESS_URL="http://istio-ingressgateway.istio-system"
+# reviews_v3_traffic_percentage
+# gets the % of productpage requests with reviews from reviews:v3 service
+# TODO: maybe make this a general purpose tool
```

Comment:
```suggestion
# TODO: generalize this function and move to samples.sh so it can be reused by
#       other tests that check traffic distribution.
```